### DISCUSSION TITLE
blog/typescript-decorators

### DISCUSSION THREAD

**Discussion by giscus** [angular-experts-io/angular-experts-website-comments#13] [General]
# blog/typescript-decorators

TypeScript five has just been released. In this release, TypeScript has implemented the new upcoming ECMA script decorators standard. Let’s take a look.

https://angularexperts.io/blog/typescript-decorators

**Comment by ilirbeq** [comment]
Nice explanation Kevin. Just add @bound decorator to the greet() method - I needed to decorate method in order for bind to work.
